pytest_mock.py

```diff
@@ -137,3 +137,84 @@ def mock(mocker):
     warnings.warn('"mock" fixture has been deprecated, use "mocker" instead',
                   DeprecationWarning)
     return mocker
+
+
+_mock_module_patches = []
+_mock_module_originals = {}
+
+
+def assert_wrapper(method, *args, **kwargs):
+    __tracebackhide__ = True
+    try:
+        method(*args, **kwargs)
+    except AssertionError as e:
+        raise AssertionError(*e.args)
+
+
+def wrap_assert_not_called(*args, **kwargs):
+    __tracebackhide__ = True
+    assert_wrapper(_mock_module_originals["assert_not_called"],
```

Comment:
Oh you are right, I was under the impression that having a local variable named `__tracebackhide__` would strip all frames below and including that frame, but it only hides the frame containing the variable so you have to raise a new exception at that point. Got it, thanks! :smile: